README.md

```diff
@@ -87,6 +90,50 @@ console.log(gen.roots) // contains the current roots nodes
 See [mafintosh/flat-tree](https://github.com/mafintosh/flat-tree) for more information about
 how node/parent indexes are calculated.
 
+## Closing up the tree and enforcing a single root
+
+If you write an uneven number of nodes to the stream, the tree will have
+dangling nodes upon further examination.
+
+```js
+const stream = new MerkleTreeStream(/* ... */)
+stream.write('a')
+stream.write('b')
+stream.write('c')
+stream.write('d')
+stream.write('e')
+```
+
+This would cause a tree structure looking like this with the `e` node dangling.
+
+```
+      abcd
+     /    \
+  ab       cd
+ /  \     /  \
+a    b   c    d   e
+```
+
+By writing the special object `MerkleTreeStream.CLOSE_UP`, you can fill
+the dangling verification nodes.
+
+```js
+stream.write(MerkleTreeStream.CLOSE_UP)
+```
+
+```
+           abcdeeee
+         /          \
+     abcd            eeee
+    /    \          /    \
+  ab      cd      **      ee
+ /  \    /  \    /  \    /  \
+a    b  c    d  *    *  *    e
+```
+
+Note that there the `*` nodes are not actually written to the merkle tree.
+If you iterate over the merkle tree structure you will not find these nodes!
+
```

Comment:
can you explain this a bit more? it will add gaps to the tree and how are those verified etc?

Reply:
Is https://github.com/mafintosh/merkle-tree-stream/pull/9/commits/f4ee9a990e20c4194b08e8246511bddfc3a25289 good? I read over the text a few times now and this makes most sense _to me_. 😅